debug: false
img_threshold: 225
window_height: 0.25
window_width: 0.85
x_pos: 0.86
y_pos: 0.5
